src/js/utils.js

```diff
@@ -16,6 +16,27 @@ exports.is_sample_genetically_altered = function is_sample_genetically_altered(d
   || datum.protein !== undefined;
 };
 
+exports.makeD3SVGElement = function(tag) {
```

Comment:
Gotcha. Have you considered passing in configuration parameters for creating a shape as in done elsewhere in the code?